.travis.yml

```diff
@@ -7,10 +7,9 @@ os:
   - osx
 
 node_js:
-  - 6
   - 8
   - 10
-  - node
```

Comment:
Generally we should test 8, 10, 12 and node. Because we want to fail early =)

Reply:
Because 12 is also LTS now. And we have many reports of users who say that it does not work on NodeJS 12.

Reply:
The only issue that I see is node-sass. Please do not add further changes to this branch, otherwise we have to rebase and merge other PRs. We should resolve the failing build in develop and not do this "hack" here =) Keep the branch as it is atm =) (before your changes)